src/analytics/core/core.interface.ts

```diff
@@ -4,8 +4,9 @@ export interface AppConfigInterface {
 }
 
 export interface OptionsInterface {
-  disableGa4Tracking?: boolean;
-  disableUaTracking?: boolean;
-  cookieDomain?: string;
+  enableGa4Tracking: boolean;
```

Comment:
I'd be tempted to list all these as optional because we can't know if the user is going to provide values for them. It will help Typescript to tell us when we're assuming they've been provided without checking them first.